### ISSUE TITLE
BUG: Kundera Cassandra CQL still requires thrift port to be open

### ISSUE THREAD

**Issue by hkroger** [Impetus/kundera#966]
When looking CassandraSchemaManager.initiateClient it seems that native protocol is not being used, only thrift, even for CQL.

Since 2.2 (or was it 3.0?), thrift port is not even open by default because it is being phased out.

**Comment by karthikprasad13** [comment]
@hkroger 

Yes, thrift port is being phased out and we are planning it in our future releases.

_Let us know if you want to work on this with us. Also check out [contribution guidelines](https://github.com/impetus-opensource/Kundera/wiki/How-to-Contribute#contribution-guidelines)._


-Karthik
